readme.md

```diff
@@ -629,6 +629,18 @@ In Codenames, two teams compete to see who can make contact with all of their ag
 | ------: | -------: | ---: |
 |   2 - 8 |       14 |  15m |
 
+### [SuperFight](https://www.superfightgame.com/)
+
+>A game of absurd arguments
```

Comment:
The [remaining lint issue](https://travis-ci.org/edm00se/awesome-board-games/builds/595689090#L233) cites needing a space between the `>` character and the start of the text.

```suggestion
> A game of absurd arguments
```